golang/examples/vout.go

```diff
@@ -7,9 +7,9 @@ import (
 	btcspv "github.com/summa-tx/bitcoin-spv/golang/btcspv"
 )
 
-func prettifyOutput(numOutput int, outpoint []byte, value uint, outputType btcspv.OutputType) string {
+func prettifyOutput(numOutput int, outpoint []byte, value uint, outputType string) string {
```

Comment:
don't change the `outputType` type here, keep it as `btcspv.OutputType`

Reply:
Oh, yes, that should be done in the prettify functions